playground/src/code-editor-test.vue

```diff
@@ -0,0 +1,194 @@
+
+const javascriptString = `function getComponents(){
+
+}`
+const typescriptString = `function get(value: string) {
+
+}`
+
+const minimapString = `{
+    "test": [], 
+    "test1": [], 
+    "test2": [], 
+    "test3": [], 
+    "test4": [], 
+    "test5": []
+}
+`
+
+const diffString = `{
+    "test": [], 
+    "test1": [], 
+    "test2": [], 
+    "test3": [], 
+    "test4": [], 
+}
+`
+
+
+const schema = JSON.stringify({
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  $ref: '#/definitions/AttrConfig',
+  definitions: {
+    AttrConfig: {
+      required: ['token', 'host'],
+      properties: {
+        hello: {
+            type: 'integer', 
+            description: 'integer'
+        }
+      },
+      additionalProperties: false,
+      type: 'object',
+    },
+  },
+});
+
+const basicJson = `
+{
+  "hello": "world"
+}
+`
+
+</script>
+
+<template>
+  <main>
+    <div style='width: 400px; height: 200px;'>
+      <v-code-editor
+        data-testid='code-editor-json-valid'
+        value='{"component": "JSON"}'
+        language='json'
+      />
+    </div>
+    <div style='width: 400px; height: 200px;'>
+      <v-code-editor
+        data-testid='code-editor-json-invalid'
+        value='{"component"}'
+        language='json'
+      />
+    </div>
+    <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-javascript-valid'
+          :value="javascriptString"
+          language='javascript'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-javascript-invalid'
+          value="def madeIt()"
+          language='javascript'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-typescript-valid'
+          :value="typescriptString"
+          language='typescript'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-typescript-invalid'
+          value="def help()"
+          language='typescript'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-python-valid'
+          value="def help()"
+          language='python'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-python-invalid'
+          value="const x = () => {}"
+          language='python'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-golang-valid'
+          value="func (s *Custom) help(x int)"
+          language='go'
+        />
+      </div>
+      <div style='width: 400px; height: 200px;'>
+        <v-code-editor
+          data-testid='code-editor-golang-invalid'
+          value="def help(x int)"
```

Comment:
this didn't actually render anything syntactically wrong on the html

Reply:
I would check with @micheal-parks on this, it looks like he built the prime component.

Reply:
More than likely we are only doing validation for JSON based on the workers we are including. We may need to include further validation logic from monaco, or remove validation checks for non-JSON editors.

Reply:
so  i see that we define language like this: 
`export let language: MonacoSupportedLanguages;`
where: 
```
export type MonacoSupportedLanguages =
| 'json'
| 'javascript'
| 'typescript'
| 'python'
| 'go'
| 'shell';
```

so could be something to do with the way that we pass in the go language? I do see on the FE when I manually write bad golang in the code samples tab there is also no invalid syntax highlighting